include/zephyr/drivers/clock_control/nrf_clock_control.h

```diff
@@ -343,6 +342,59 @@ void nrf_clock_control_hfxo_release(void);
 
 #endif /* defined(CONFIG_CLOCK_CONTROL_NRF2) */
 
+/**
+ * @brief Set the audiopll's frequency fraction.
+ *
+ * @param freq_fraction Frequency fractional divider ratio 0-65535 to set.
+ *
+ * @note Use smaller values (under 2000) to apply fine tuning on audiopll,
+ * otherwise new base frequency will be applied resulting in going out of
+ * lock statefor some time.
```

Comment:
There should be some form of callback if the PLL looses lock.

Reply:
There is none in the NRFS service, so there is nothing to attach the callback to